### ISSUE TITLE
Watch examples in Readme.md are wrong

### ISSUE THREAD

**Issue by saamalik** [ahmetb/kubectl-aliases#40]
The README has the following examples:
```
alias kgsvcwn='watch kubectl get service --namespace'
alias kgsvcslwn='watch kubectl get service --show-labels --namespace'
alias kgwf='watch kubectl get -f'
```

However, in actuality the watch commands is:
```
$ type kgwf
kgwf is aliased to `kubectl get --watch --recursive -f'
```

**Comment by ahmetb** [comment]
Ah, shit. I was doing:

    [ -f ~/.kubectl_aliases ] && source \
		<(cat ~/.kubectl_aliases | sed -r 's/(kubectl.*) --watch/watch\1/g')

Maybe I should add this to the README. 
I personally find kubectl's --watch very useless. so I did this hack to switch `--watch` to the beginning of the command as `watch `.

**Comment by ahmetb** [comment]
fixed in d4c5c6a.
